src/components/details.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+import Navbar from './Navbar';
+import '../styles/Details.css';
+
+const MyDetails = () => {
+  const [company, setCompany] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const params = useParams();
+  const { companyName } = params;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const response = await axios.get(`https://588fc30f7458d612002df0d2.mockapi.io/api/v1/companies?filter=${companyName}`);
+        setCompany(response.data);
+      } catch (error) {
+        console.error(error);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [companyName]);
+
+  return (
+    <div className="details-container">
+      <Navbar />
+      {isLoading ? (
+        <p className="load">Loading...</p>
+      ) : (
+        company.map((company) => (
+          <div key={company.id} className="details-content">
+            <div className="company-name">
```

Comment:
- [ ] Kindly you might consider sticking to the original design, as much as possible. the structure, spacing, and ... must look similar. Kindly adjust your details page to look similar to the template.
as you may see, we have a structure like:
- A navbar on the top
- Name of the company 
- The details about the company
### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233450931-f181d3cc-7af4-434c-b37d-891197d4e753.png)
### Original template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233451123-8ad8f105-7d2e-4792-b040-ebd1498adee8.png)